assets/js/index.js

```diff
@@ -0,0 +1,458 @@
+  // create section
+  const section = document.createElement("section");
+  section.setAttribute("class", "content-section question-section");
+  section.setAttribute("id", "question-container");
+  // creat h2
+  const h2 = document.createElement("h2");
+  h2.setAttribute(
+    "class",
+    "question-section-title animate__animated animate__bounceInDown"
+  );
+  // specific to the question (dynamic)
+  h2.textContent = `${questionIndex + 1}. ${currentQuestion.text}`;
+  // creat ul and append 4 li
+  const ul = document.createElement("ul");
+  ul.setAttribute(
+    "class",
+    "feedback-list animate__animated animate__bounceInDown"
+  );
+
+  // call function to create and append list items to ul****
+  const li1 = document.createElement("li");
+  li1.setAttribute("class", "list-item");
+  li1.setAttribute("data-value", currentQuestion.options[0]);
+  li1.textContent = currentQuestion.options[0];
+  const li2 = document.createElement("li");
+  li2.setAttribute("class", "list-item");
+  li2.setAttribute("data-value", currentQuestion.options[1]);
+  li2.textContent = currentQuestion.options[1];
+  const li3 = document.createElement("li");
+  li3.setAttribute("class", "list-item");
+  li3.setAttribute("data-value", currentQuestion.options[2]);
+  li3.textContent = currentQuestion.options[2];
+  const li4 = document.createElement("li");
+  li4.setAttribute("class", "list-item");
+  li4.setAttribute("data-value", currentQuestion.options[3]);
+  li4.textContent = currentQuestion.options[3];
+  ul.append(li1, li2, li3, li4);
+
+  //append h2, ul and div to section
+  section.append(h2, ul);
+
+  // append question section to main element
+  main.append(section);
+  //add event listener on question section
+  section.addEventListener("click", handleOptionClick);
+};
+
+// function to render timer
+const renderTimerSection = () => {
+  // use HTML as guide and build in JS
+  const timerSection = document.createElement("section");
+  timerSection.setAttribute("class", "timer-section");
+  timerSection.setAttribute("id", "timer");
+  const timeRemaining = document.createElement("div");
+  timeRemaining.setAttribute("class", "timer");
+  const timeSpan = document.createElement("span");
+  timeSpan.setAttribute("id", "time-span");
+  timeSpan.setAttribute("class", "time-span-class");
+
+  //append span to time remaining then time remaining to section
+  timeRemaining.appendChild(timeSpan);
+  timerSection.append(timeRemaining);
+  main.append(timerSection);
+
+  // function for timer countdown
+  const timerCountdownUpdate = () => {
+    //reduce time
+    timeCountdown -= 1;
+    timeSpan.textContent = `Time Remaining: ${timeCountdown} seconds`;
+    if (timeCountdown === 0) {
+      clearInterval(timerId);
+    }
+  };
+  const timerId = setInterval(timerCountdownUpdate, 1000);
+};
+
+//function to banner section
+const removeBanner = () => {
+  banner.remove();
+};
+// function to remover question section
+const removeQuestion = () => {
```

Comment:
Since this function is only used once, and it only contains one line of code, it isn't really necessary to put it in a separate function. It would make more sense just to use `document.getElementById("question-container").remove();` directly where it is needed, rather than create an extra unnecessary function.